### DISCUSSION TITLE
Power Consumption

### DISCUSSION THREAD

**Discussion by Qmick01** [cyberman54/ESP32-Paxcounter#987] [General]
Hi All,

I have been running the Pax counter on the TTGO LORA OLED board using Lipo battery packs. 
A 20Ah pack lasts about 30 days with a 600 second sleep cycle and BLE only enabled. Additionally the OLED is off and the onboard LEDs are all removed. 

I don't think the TTGO board is the best for my uses as I have seen commercial LORA devices which sample things and contain an 8 or 9 ah battery and have a 10 year life span even though they may send data every hour. 

Has anyone spent any time testing the Pax counter code on different boards to get maximum power efficiency or have any other tips to extend battery life?

Thanks in advance.

**Comment by cyberman54** [comment]
It's not the board hardware, it's the application. Wifi and/or BLE sniffing draws far more power than all other board components. Thus, it makes no sense to compare a wifi/BLE sniffer with a simple sensor LORA application.

Nevertheless, there are some boards with lower power consumption in sleep mode, than TTGO 1.6.1. But this won't leverage lifetime of a battery powered sniffing application. The relevant impact comes from sniffing duration and dutycycle.

**Reply by Qmick01** [reply]
Thanks. 
The present configuration uses two transmit cycles, one for battery and one for PAX. Would reducing that to a single transmission save power?

**Reply by cyberman54** [reply]
again, the power consumption comes not from transmit cycles, but from sniffing duration.

**Comment by Qmick01** [comment]
Thanks for your reply.

I think the sniffing cycle I have used is 30 seconds. If I shorten that
period does it have an impact on the results or the connection process? I
seem to recall I had tried it on a 15 second cycle but then the devices
couldn't reconnect if they lost contact with a gateway.
